README.md

```diff
@@ -1,2 +1,11 @@
+# travis version build  
+[![Build Status](https://travis-ci.com/joszo/kakunin-cli.svg?branch=next)](https://travis-ci.org/TheSoftwareHouse/kakunin-cli.svg?branch=master)
```

Comment:
You sure it's getting it from tsh travis? It looks like you're pointing it to "next" branch on your travis

Reply:
changed -> [![Build Status](https://travis-ci.org/TheSoftwareHouse/kakunin-cli.svg?branch=master)](https://travis-ci.org/TheSoftwareHouse/kakunin-cli.svg?branch=master)